settings.py

```diff
@@ -323,12 +324,16 @@
     'openid'
 ]
 
+SOCIAL_AUTH_PROVIDER = 'google-oauth2'
```

Comment:
what does this do and why didn't we need it before?

Reply:
Before, the string `'google-oauth2'` is used directly in some places. It is not so straightforward for purpose of the string itself. Defining a constant with a name makes the code more readable.